contracts/Uniswap/UniswapAnchoredView.sol

```diff
@@ -1,32 +1,32 @@
 // SPDX-License-Identifier: GPL-3.0
 
-pragma solidity ^0.6.10;
+pragma solidity ^0.6.12;
 pragma experimental ABIEncoderV2;
 
-import "../OpenOraclePriceData.sol";
 import "./UniswapConfig.sol";
 import "./UniswapLib.sol";
+import "../Ownable.sol";
+import "../Chainlink/AggregatorValidatorInterface.sol";
 
 struct Observation {
     uint timestamp;
     uint acc;
 }
 
-contract UniswapAnchoredView is UniswapConfig {
-    using FixedPoint for *;
+struct PriceData {
+    uint248 price;
+    bool failoverActive;
+}
 
-    /// @notice The Open Oracle Price Data contract
-    OpenOraclePriceData public immutable priceData;
+contract UniswapAnchoredView is AggregatorValidatorInterface, UniswapConfig, Ownable {
+    using FixedPoint for *;
```

Comment:
Any way we can restrict the type here?

Reply:
We can, but this is an unchanged line that git seems to have deemed changed (see removed line 16 above). `using FixedPoint for *;` is used in a few contracts throughout the repo.